src/OSPSuite.InstallationValidator.Core/Services/BatchComparisonTask.cs

```diff
@@ -2,16 +2,16 @@
 using System.Linq;
 using System.Threading;
 using System.Threading.Tasks;
-using OSPSuite.Core.Domain;
+using FluentNHibernate.Utils;
 using OSPSuite.InstallationValidator.Core.Assets;
 using OSPSuite.InstallationValidator.Core.Domain;
 
 namespace OSPSuite.InstallationValidator.Core.Services
 {
    public interface IBatchComparisonTask
    {
-      Task<BatchComparisonResult> StartComparison(string folderPath, CancellationToken token);
-      Task<BatchComparisonResult> StartComparison(string folderPath1, string folderPath2, CancellationToken token);
+      Task<BatchComparisonResult> StartComparison(string folderPath, CancellationToken token, string folderPathCaption1 = "Old", string folderPathCaption2 = "New");
+      Task<BatchComparisonResult> StartComparison(string folderPath1, string folderPath2, CancellationToken token, string folderPathCaption1 = "Old", string folderPathCaption2 = "New");
```

Comment:
Define constants for that (const string sot that they can be used as default parameters )